datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -493,6 +499,7 @@ async fn fetch_statistics(
 pub fn statistics_from_parquet_meta_calc(
     metadata: &ParquetMetaData,
     table_schema: SchemaRef,
+    force_string_view: bool,
```

Comment:
This is not super elegant as we need to pass this config deep into the implementations. But I don't have a better way to do this.